### PR TITLE
Feat/renovate

### PR DESCRIPTION
**Pull request by DerekRoberts**
Test out Mend Rennovate to batch Dependabot-type updates together, reducing noise and effort.

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-quickstart-typescript-790-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-quickstart-typescript-790-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-quickstart-typescript/actions/workflows/merge-main.yml)

### CODE REVIEW

File: .github/workflows/renovate.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - ".github/ISSUE_TEMPLATE/*"
+      - "**.md"
+  schedule:
+    # Quotes required for "*" (#42, asterisk), special semantics in YAML
+    - cron: "0/15 * * * *"
```
Comment (paulushcgcj) [inline .github/workflows/renovate.yml:11]: Is this running every 15 minutes? It seems kind of a lot. It can be executed once a day `"0 0 * * *"`

**Review by paulushcgcj — CHANGES_REQUESTED** [review]
With that do you need to change the [dependabot](.github/dependabot.yml#L8) file?

**Comment by DerekRoberts** [comment]
> With that do you need to change the [dependabot](.github/dependabot.yml#L8) file?

On a regular repo we'd disable Dependabot. Here I want to run both and compare.

File: .github/workflows/renovate.yml (hunk 1 above)
Comment (DerekRoberts) [reply .github/workflows/renovate.yml:11]: Whoops! Totally correct. This was from Mend's sample.

**Review by paulushcgcj — APPROVED** [review]

**Comment by mishraomp** [comment]
LGTM, super interested to see how it works :)

**Pull request merged**